Makefile

```diff
@@ -368,15 +368,30 @@ $(CTL_CONTAINER_CREATED): calicoctl/Dockerfile.calicoctl dist/calicoctl
 	docker build -t $(CTL_CONTAINER_NAME) -f calicoctl/Dockerfile.calicoctl .
 	touch $@
 
-## Run the build in a container. Useful for CI
+## Build calicoctl
+binary: $(CALICOCTL_FILES) vendor
+	GOOS=${OS} GOARCH=${ARCH} CGO_ENABLED=0 go build -v -o dist/calicoctl-${OS}-${ARCH} $(LDFLAGS) "./calicoctl/calicoctl.go"
```

Comment:
You shouldn't need to set those.  If they're unset, they'll just default to the values at the top of the file.